packages/errors/src/context.ts

```diff
@@ -50,8 +54,21 @@ type DefaultUnspecifiedErrorContextToUndefined<T> = {
  *   - Don't change or remove members of an error's context.
  */
 export type KinobiErrorContext = DefaultUnspecifiedErrorContextToUndefined<{
-    [KINOBI_ERROR__ANCHOR__UNRECOGNIZED_IDL_TYPE]: {
+    [KINOBI_ERROR__ANCHOR__ACCOUNT_TYPE_MISSING]: {
+        name: CamelCaseString;
+    };
+    [KINOBI_ERROR__ANCHOR__ARGUMENT_TYPE_MISSING]: {
+        name: CamelCaseString;
+    };
+    [KINOBI_ERROR__ANCHOR__SEED_KIND_UNIMPLEMENTED]: {
+        kind: string;
+    };
+    [KINOBI_ERROR__ANCHOR__TYPE_PATH_MISSING]: {
         idlType: string;
+        path: string;
+    };
+    [KINOBI_ERROR__ANCHOR__UNRECOGNIZED_IDL_TYPE]: {
+        name: CamelCaseString;
```

Comment:
The context of this error type shouldn't be changed as it's already being used in the code.

```suggestion
        idlType: string;
```